File: laramie-ingresos-publicos-webapi/src/domain/entities/entidad.ts

export default class Entidad {

    id: number;
    codigo: string;
    tipo: string;
    nombre: string;
    orden: number;
    dato1: string;
    dato2: string;
    dato3: string;
    dato4: string;
    dato5: string;
    dato6: string;
    dato7: string;
    dato8: string;
    dato9: string;
    dato10: string;

    constructor(
        id: number = 0,
        codigo: string = "",
        tipo: string = "",
        nombre: string = "",
        orden: number = 0,
        dato1: string = "",
        dato2: string = "",
        dato3: string = "",
        dato4: string = "",
        dato5: string = "",
        dato6: string = "",
        dato7: string = "",
        dato8: string = "",
        dato9: string = "",
        dato10: string = "")
    {
        this.id = id;
        this.codigo = codigo;
        this.tipo = tipo;
        this.nombre = nombre;
        this.orden = orden;
        this.dato1 = dato1;
        this.dato2 = dato2;
        this.dato3 = dato3;
        this.dato4 = dato4;
        this.dato5 = dato5;
        this.dato6 = dato6;
        this.dato7 = dato7;
        this.dato8 = dato8;
        this.dato9 = dato9;
        this.dato10 = dato10;
    }

    setFromObject = (row) =>
    {
        this.id = row.id ?? 0;
        this.codigo = row.codigo ?? "";
        this.tipo = row.tipo ?? "";
        this.nombre = row.nombre ?? "";
        this.orden = row.orden ?? 0;
        this.dato1 = row.dato1 ?? "";
        this.dato2 = row.dato2 ?? "";
        this.dato3 = row.dato3 ?? "";
        this.dato4 = row.dato4 ?? "";
        this.dato5 = row.dato5 ?? "";
        this.dato6 = row.dato6 ?? "";
        this.dato7 = row.dato7 ?? "";
        this.dato8 = row.dato8 ?? "";
        this.dato9 = row.dato9 ?? "";
        this.dato10 = row.dato10 ?? "";
    }
    
}

File: modules/tesoreria/models/entidad.py
from sqlalchemy import Column, BigInteger, String, Integer

import sys, os
sys.path.insert(0, os.path.join(os.path.dirname(__file__), "..", ".."))
from shared.database import Base


class Entidad(Base):
    __tablename__ = "tesoreria_entidades"

    id = Column(BigInteger, primary_key=True, autoincrement=True)
    codigo = Column(String(50), nullable=False)
    tipo = Column(String(50), nullable=False, index=True)
    nombre = Column(String(250), nullable=False)
    orden = Column(Integer, nullable=False, default=0)
    dato1 = Column(String(250), nullable=True)
    dato2 = Column(String(250), nullable=True)
    dato3 = Column(String(250), nullable=True)
    dato4 = Column(String(250), nullable=True)
    dato5 = Column(String(250), nullable=True)
    dato6 = Column(String(250), nullable=True)
    dato7 = Column(String(250), nullable=True)
    dato8 = Column(String(250), nullable=True)
    dato9 = Column(String(250), nullable=True)
    dato10 = Column(String(250), nullable=True)
